plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -1,14 +1,33 @@
 {% macro snowflake__create_table_as(temporary, relation, sql) -%}
   {%- set transient = config.get('transient', default=true) -%}
+  {%- set cluster_by_keys = config.get('cluster_by', default=none) -%}
+  {%- set enable_automatic_clustering = config.get('automatic_clustering', default=false) -%}
+  {%- if cluster_by_keys is not none and cluster_by_keys is string -%}
+    {%-  set cluster_by_keys = [cluster_by_keys] -%}
+    {%- set cluster_by_string = cluster_by_keys|join(", ")-%}
```

Comment:
Just did one last scan here - I think this won't work properly if `cluster_by_keys` is provided as a list. If that happens, then the code in this branch won't be executed. I think we should move this line outside of the if block here.

Reply:
ohh dear! good catch! I can certainly fix that! BUT I just realised to cleaned my fork so I don't actually know how to can edit this on my side. I had to fork dbt again as we were experimenting and forgot this was still pending. Do you know what would be the best way to go? I could probably fork again, or do a new PR but then we'd loose the trace of our discussion (although we could refer to it even if the PR is closed). Let me know

Reply:
oh dear indeed! I've never seen this before!

There's a cool thing you can do on GitHub - if you add `.patch` to the end of a PR url, you'll get a "patch" for the change. I think you should be able to make a clean branch off of `dev/0.14.1`, then do something like:

```
curl https://patch-diff.githubusercontent.com/raw/fishtown-analytics/dbt/pull/1591.patch > 1591.patch
git apply 1591.patch
```

If git says that it can't apply the patch, you can try applying the patch from the commit `9e36ebd`.

Let me know how it goes!